content/docs/dvclive/user-guide/ml-frameworks/index.md

```diff
@@ -0,0 +1,17 @@
+# ML Frameworks
+
+DVCLive makes it easy to add experiment tracking capabilities to your existing
+_ML training code_.
+
+We currently support the following _ML Frameworks_:
+
+- [Keras](/docs/dvclive/user-guide/ml-frameworks/keras)
+- [MMCV](/docs/dvclive/user-guide/ml-frameworks/mmcv)
+- [PyTorch](/docs/dvclive/user-guide/ml-frameworks/pytorch)
+- [TensorFlow](/docs/dvclive/user-guide/ml-frameworks/tensorflow)
+- [XGBoost](/docs/dvclive/user-guide/ml-frameworks/xgboost)
+
+The DVCLive team is happy to extend the integrations as needed. Please check the
```

Comment:
The wording here and throughout the ML framework pages still seems inconsistent and unclear. We still call them all "integrations" here and elsewhere.

Reply:
Removed